ui/app/templates/components/json-editor.hbs

```diff
@@ -20,17 +20,17 @@
 		</div>
 	{{/if}}
 
-	{{ivy-codemirror
-		data-test-component="json-editor" 
-		value=@value
-		options=this.options
-		valueUpdated=(action "updateValue")
-		onFocusOut=(action "onFocus")    
-	}}
+	<IvyCodemirror
```

Comment:
Not something for you to fix here, but the indentation looks off on this file. I think I've dealt with it too, and I'm not sure what's up with this particular file.

Reply:
Weird it looks good in my editor. I think it would be a good idea to use the prettier plugin for ember-template-lint as well to catch these things. I'll bring it up in our discussion.
https://github.com/ember-template-lint/ember-template-lint-plugin-prettier

Reply:
@Monkeychip there is a _stylistic_ rule set for ember-template-lint which I enabled and it found this identation issue among many others. I think it's a good idea to enforce it.